pkg/logrus/proxy.go

```diff
@@ -196,113 +218,133 @@ func (p *Proxy) Panicln(args ...any) {
+
+func Print(args ...any) {
+	proxy.Load().Print(args...)
 }
 
 func Trace(args ...any) {
-	proxy.Trace(args...)
+	proxy.Load().Trace(args...)
 }
 
 func Debug(args ...any) {
-	proxy.Debug(args...)
+	proxy.Load().Debug(args...)
 }
 
 func Info(args ...any) {
-	proxy.Info(args...)
+	proxy.Load().Info(args...)
 }
 
 func Warn(args ...any) {
-	proxy.Warn(args...)
+	proxy.Load().Warn(args...)
+}
+
+func Warning(args ...any) {
+	proxy.Load().Warning(args...)
 }
 
 func Error(args ...any) {
-	proxy.Error(args...)
+	proxy.Load().Error(args...)
 }
 
 func Fatal(args ...any) {
-	proxy.Fatal(args...)
+	proxy.Load().Fatal(args...)
 }
 
 func Panic(args ...any) {
-	proxy.Panic(args...)
+	proxy.Load().Panic(args...)
+}
+
+func Printf(format string, args ...any) {
+	proxy.Load().Printf(format, args...)
 }
 
 func Tracef(format string, args ...any) {
-	proxy.Tracef(format, args...)
+	proxy.Load().Tracef(format, args...)
 }
 
 func Debugf(format string, args ...any) {
-	proxy.Debugf(format, args...)
+	proxy.Load().Debugf(format, args...)
 }
 
 func Infof(format string, args ...any) {
-	proxy.Infof(format, args...)
+	proxy.Load().Infof(format, args...)
 }
 
 func Warnf(format string, args ...any) {
-	proxy.Warnf(format, args...)
+	proxy.Load().Warnf(format, args...)
 }
 
 func Warningf(format string, args ...any) {
-	proxy.Warnf(format, args...)
+	proxy.Load().Warnf(format, args...)
 }
 
 func Errorf(format string, args ...any) {
-	proxy.Errorf(format, args...)
+	proxy.Load().Errorf(format, args...)
 }
 
 func Fatalf(format string, args ...any) {
-	proxy.Fatalf(format, args...)
+	proxy.Load().Fatalf(format, args...)
 }
 
 func Panicf(format string, args ...any) {
-	proxy.Panicf(format, args...)
+	proxy.Load().Panicf(format, args...)
 }
 
 func WithField(key string, value any) *Proxy {
-	return proxy.WithField(key, value)
+	return proxy.Load().WithField(key, value)
 }
 
 func WithFields(fields Fields) *Proxy {
-	return proxy.WithFields(fields)
+	return proxy.Load().WithFields(fields)
+}
+
+func WithError(err error) *Proxy {
+	return proxy.Load().WithError(err)
 }
 
 func WithContext(ctx context.Context) *Proxy {
-	return proxy.WithContext(ctx)
+	return proxy.Load().WithContext(ctx)
+}
+
+func Println(args ...any) {
+	proxy.Load().Println(args...)
 }
 
 func Traceln(args ...any) {
-	proxy.Traceln(args...)
+	proxy.Load().Traceln(args...)
 }
 
 func Debugln(args ...any) {
-	proxy.Debugln(args...)
+	proxy.Load().Debugln(args...)
 }
 
 func Infoln(args ...any) {
-	proxy.Infoln(args...)
+	proxy.Load().Infoln(args...)
 }
 
 func Warnln(args ...any) {
-	proxy.Warnln(args...)
+	proxy.Load().Warnln(args...)
 }
 
 func Warningln(args ...any) {
-	proxy.Warningln(args...)
+	proxy.Load().Warningln(args...)
 }
 
 func Errorln(args ...any) {
-	proxy.Errorln(args...)
+	proxy.Load().Errorln(args...)
 }
 
 func Fatalln(args ...any) {
-	proxy.Fatalln(args...)
+	proxy.Load().Fatalln(args...)
 }
 
 func Panicln(args ...any) {
-	proxy.Panicln(args...)
+	proxy.Load().Panicln(args...)
 }
```

Comment:
I did not encounter any thread-safety issues, but since this is a good practice to do and `slog` library itself uses the same technique, why not to use it.